src/routes/note-create.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import ReactRouterPropTypes from 'react-router-prop-types';
+import { Redirect } from 'react-router';
+
+import { NoteCreatePage } from '@folio/stripes/smart-components';
+
+import { entityTypesTranslationKeys } from '../constants';
+
+export default class NoteCreateRoute extends Component {
+  static propTypes = {
+    history: PropTypes.shape({
+      goBack: PropTypes.func.isRequired,
+    }).isRequired,
+    location: ReactRouterPropTypes.location.isRequired,
+  };
+
+  getReferredEntityData() {
+    const {
+      entityName: name,
+      entityType: type,
+      entityId: id,
```

Comment:
Please update the readme for the connectet component to reflect the change from the queryparams to location state

Reply:
@Godlevskyi, the connected component receives referred entity data as a prop. Query params or location.state are used in apps' routes